src/fundamental/edid-fundamental.c

```diff
@@ -0,0 +1,47 @@
+/* SPDX-License-Identifier: LGPL-2.1-or-later */
+
+#include "edid-fundamental.h"
+
+#define EDID_FIXED_HEADER_PATTERN "\x00\xFF\xFF\xFF\xFF\xFF\xFF"
+assert_cc(sizeof_field(EdidHeader, pattern) == sizeof(EDID_FIXED_HEADER_PATTERN));
+
+bool edid_parse_blob(const void *blob, size_t blob_size, EdidHeader *ret_header) {
+        assert(ret_header);
+        /* EDID size is at least 128 as per the specification */
+        if (blob_size < 128)
+                return false;
```

Comment:
This violates our coding style: always initialize `ret_xyz` on success (>= 0).
So, please return a negative errno e.g. -EINVAL, or initialize `ret_header` with zero.
In this case, returning `-EINVAL` is better, I think.